src/main/resources/assets/extended_industrialization/lang/ru_ru.json

```
@@ -0,0 +1,322 @@
+  "text.extended_industrialization.electric_tool_help_2_fortune_silk_touch": "- Нажмите %s + %s, чтобы переключаться между Удачей и Шёлковым касанием.",
+  "text.extended_industrialization.electric_tool_help_2_looting_beheading": "- Нажмите %s + %s, чтобы переключаться между Добычей и Обезглавливанием.",
+  "text.extended_industrialization.electric_tool_help_3": "- Используйте %s + %s, чтобы менять скорость добычи.",
+  "text.extended_industrialization.electric_tool_help_4": "- Нажмите %s, удерживая предмет, или %s при наведении, чтобы переключить добычу 3×3.",
+  "text.extended_industrialization.enchantment_module_input": "Вставьте модуль зачарования, чтобы машина использовала зачарование.",
+  "text.extended_industrialization.enchantment_module_machine": "Можно использовать в %s.",
+  "text.extended_industrialization.enchantment_module_single_value": "Применяет %s в машине на уровне %s.",
+  "text.extended_industrialization.enchantment_module_values": "Напряжение определяет уровень %s, применяемый в машине.",
+  "text.extended_industrialization.farmer_not_tilling": "Не вспахивает",
+  "text.extended_industrialization.farmer_planting_alternating_lines": "Чередующиеся линии",
+  "text.extended_industrialization.farmer_planting_as_needed": "По необходимости",
+  "text.extended_industrialization.farmer_planting_quadrants": "Квадранты",
+  "text.extended_industrialization.farmer_task": "  - %s: %s",
+  "text.extended_industrialization.farmer_task_fertilizing": "Удобрение",
+  "text.extended_industrialization.farmer_task_fertilizing_description": "При подаче подходящего жидкого удобрения случайно удобряет культуры и саженцы.",
+  "text.extended_industrialization.farmer_task_harvesting": "Сбор",
+  "text.extended_industrialization.farmer_task_harvesting_description": "При наличии достаточного места для вывода собирает полностью выросшие культуры и деревья.",
+  "text.extended_industrialization.farmer_task_hydrating": "Увлажнение",
+  "text.extended_industrialization.farmer_task_hydrating_description": "При подаче воды вспаханная почва будет увлажнена.",
+  "text.extended_industrialization.farmer_task_planting": "Посадка",
+  "text.extended_industrialization.farmer_task_planting_description": "При подаче культур или саженцев высаживает их на подходящую почву. Разные режимы посадки формируют разные схемы.",
+  "text.extended_industrialization.farmer_task_tilling": "Вспашка",
+  "text.extended_industrialization.farmer_task_tilling_description": "При включении блоки земли превращаются в пашню. Не работает без подачи воды.",
+  "text.extended_industrialization.farmer_task_tooltip": "Может выполнять следующие задачи с помощью %s:",
+  "text.extended_industrialization.farmer_tilling": "Вспашка",
+  "text.extended_industrialization.fluid_fertilizers": "Жидкие удобрения",
+  "text.extended_industrialization.fluid_fertilizers_consumes": "Потребление: %d мб",
```

мб замени на [мВ](https://github.com/AztechMC/Modern-Industrialization/blob/1.21.x/src/main/resources/assets/modern_industrialization/lang/ru_ru.json#L1489)
```suggestion
  "text.extended_industrialization.fluid_fertilizers_consumes": "Потребление: %d мВ",
```